Formula/s/stellar-cli.rb

```diff
@@ -0,0 +1,32 @@
+class StellarCli < Formula
+  desc "Stellar command-line tool for interacting with the Stellar network"
+  homepage "https://developers.stellar.org"
+  url "https://github.com/stellar/stellar-cli.git",
+      tag:      "v21.3.0",
+      revision: "11746937e813346272faf68e70cbdbc1b0df780f"
+  license "Apache-2.0"
+  head "https://github.com/stellar/stellar-cli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Do we really need this? Does the default `livecheck` strategy (when this block is omitted) not work correctly?

Reply:
I'll remove it if it's covered by a default. I included it because I saw it in some other existing formulas.

Reply:
```suggestion
```